docs/DeveloperGuide.md

```diff
@@ -626,36 +684,66 @@ Preconditions: The contact the user wishes to edit is displayed on the UI.
 
     Use case ends.
     
-#### **Use case: Add a tag**
+
+#### **Use case: Add a reminder**
 
 **MSS**
 
-1. User requests to add a tag.
+1. User requests to add a reminder.
 
-1. User specifies contacts to be added to this tag.
+1. User specifies which event this reminder is for.
 
-1. User specifies other tags to be added as child-tags.
+1. User specifies when he wants to activate this reminder.
 
-1. Athena adds the new tag and its relations to specified contacts and child-tags.
+1. Athena adds the new reminder.
 
     Use case ends.
 
 **Extensions**
-* 1a. The tag already exists in Athena.
+* 2a. There is already a reminder for the event.
 
     * 1a1. Athena shows an error message.
     
       Use case ends.
-* 2a. Contact specified is invalid.
+* 2a. The target event does not exist. Event index entered is negative or greater than the size of the events list.
 
     * 2a1. Athena shows an error message.
     
       Use case ends.
-* 3a. Child-tag specified does not exist.
+* 3a. The reminder is set to activate before the current date.
 
     * 3a1. Athena shows an error message.
     
       Use case ends.
```

Comment:
uhhh why's the add tag use case being written over?

Reply:
reverted